src/main/java/com/github/bordertech/utils/Symbols.java

```diff
@@ -32,7 +38,34 @@
 			{"\u2039", "<"},
 			{"\u203A", ">"},
 			{"\u02DC", "~"},
-			{"\u00A0", " "}
+			{"\u0020", " "},
```

Comment:
@sroesler I have to ask, what is the point of this one?

Reply:
Apologies, this is an oversight, will remove...